### ISSUE TITLE
Update Web Lit Basics I activity pages

### ISSUE THREAD

**Issue by chadsansing** [mozilla/learning.mozilla.org#1876]
@hannahkane, @KGorr, & @ldecoursy, let's do this.

First, we could use a quick review of
- https://chadsansing.github.io/curriculum-testing/updated-web-lit-basics/session01-kraken-the-code.html
- https://chadsansing.github.io/curriculum-testing/updated-web-lit-basics/session02-ping-kong.html
- https://chadsansing.github.io/curriculum-testing/updated-web-lit-basics/session03-hack-the-news.html
- https://chadsansing.github.io/curriculum-testing/updated-web-lit-basics/session04-html-puzzle-boxes.html
- https://chadsansing.github.io/curriculum-testing/updated-web-lit-basics/session05-web-chef.html
- https://chadsansing.github.io/curriculum-testing/updated-web-lit-basics/session06-story-of-us.html

Then we can clone https://github.com/chadsansing/curriculum-testing/updated-web-lit-basics-1/ into a Mozilla repo and link from to the updated activities indexed on https://learning.mozilla.org/activities/web-lit-basics/.

Make sense?

**Comment by ldecoursy** [comment]
For Hack the News, you can link to this updated X-Ray Goggles cheat sheet: 
* Thimble: https://thimbleprojects.org/mozillalearning/48435/ 
* Printable: https://drive.google.com/a/mozillafoundation.org/file/d/0By3iomeqwUgBM2g3T0FER2I2V3M/view

Bigger question - what does this mean for the localized versions of these activities? Currently you can select the available languages from a pull-down on the top right of the activities.

**Comment by chadsansing** [comment]
That's true - let's loop in @soapdog re: l10n!

**Comment by KGorr** [comment]
@chadsansing, my plate is full today, but am happy to help and put on my to-do list for early next week!

**Comment by hannahkane** [comment]
[This ticket](https://github.com/mozilla/learning.mozilla.org/issues/1801) is about migrating the localized content to the new templates. @chadsansing - are we making content changes in addition to template/location changes?

**Comment by chadsansing** [comment]
I probably got ahead of myself. This is lightly updated content in the new curriculum template. Let's back up and figure out what to do when :)

**Comment by hannahkane** [comment]
@chadsansing - do you think the edits are substantive enough to require re-localization? If so, we should consider the best way to present the delta to help make things easier for the localizers.

**Comment by chadsansing** [comment]
No. In most cases, I cleaned up a typo we missed or something similar. I added a line in the Story of Us about connecting via learning dot or the current instance of Discourse. That was about it.

**Comment by KGorr** [comment]
@chadsansing, can you clarify what you'd like my assistance with? 😄

**Comment by chadsansing** [comment]
@KGorr - it would be great if you could do a read through sometime soon to catch any errors I've made in copying and pasting.

**Comment by soapdog** [comment]
@KGorr @chadsansing @hannahkane should I use this version and add the localized content to it?

**Comment by chadsansing** [comment]
If that can be accomplished, my vote is yes - @KGorr, @ldecoursy?

**Comment by hannahkane** [comment]
@soapdog - yes, I think so, too

**Comment by hannahkane** [comment]
@chadsansing - tiny thing, but I was just fixing the link to the Tip Sheet in the currently-published version of Kraken the Code (as per #1878), and then I realized we might need to fix it here. But it doesn't look like you link to the Tip Sheet in this version. Is that true?

**Comment by chadsansing** [comment]
@ldecoursy - can you ask MOUSE if they have an updated tip sheet for Kraken the Code?

**Comment by ldecoursy** [comment]
Their updated tip sheet is here: https://docs.google.com/file/d/0ByDhuNZWuJn-bmdaTkoxeDZ4Vmc/edit

**Comment by KGorr** [comment]
I'm working on a read through of all of these activities as we speak. I'll post tweaks needed soon.

**Comment by KGorr** [comment]
OK, here are the things I noticed:

For all –  “Activity of 1 of 6” at the top of each homepage needs to drop the 1st “of”

**Kraken the Code**

Homepage
- 1st learning objectives bullet isn’t capped
- 4th learning objective ends with a “/”
- 5th learning objective doesn’t have ending punctuation

Kraken the Code: Hunt
- extra space in 1st sentence of 1st paragraph between the ending word and punctuation.
- extra space in 1st sentence of 2nd paragraph between the ending word and punctuation.

**Ping Kong**

Preparation
- Typo: “Labelled “ in 2nd paragraph after definitions list

Intro
- Video is not linked/working

Set Up Obstacle Course
- 1st sentence “Say that to help understand this path, we're going to act out a human ping/traceroute in an obstacle course.” Should be reworded to “Explain to the group that in order to help…”
- typo “labelled”  in 4th paragraph

Model the Obstacle Course
- “Give the learners the following scenario” should end with a “:”

**Hack the News**

Homepage
- typo “Internect” under Materials

Preparation
- “the Web X-Ray Goggles first time user activity” should say “the Web X-Ray Goggles sample activity” since that is how it is described on the website
- last sentence section “Post the URL, or Web address, of the Web X-Ray Goggles” can omit the 2nd “the” 
    - actually, global thought, my preference is that “the” is not in front of “goggles” ever – but that may differ from a style guide or branding of goggles…I started to add a bunch of notes to omit “the” but stopped in case you don’t want to!

Hack the News Using the Web X-Ray Goggles – should section instead be named “Hack the News Using X-Ray Goggles”? 
- 1st paragraph “Install the X-Ray Goggles” add ending punctuation
- “Once you've installed the goggles, try the first-time user experience linked to at the bottom of the homepage.” Change “first time user experience” to “sample activity” and omit “to” after “linked” 
- “Ask for suggestions for news site.” Should it say “…for a news site.”?

This activity doesn’t link to the next one in the series like the others do.

**HTML Puzzle Boxes**
Homepage
- Pink callout box typo “amd”

Puzzle Box Challenge
- The only instructions are “First assemble the tags in the right order in a small group. Then race!” What does race mean?

Reflections
- Should link to x-ray goggles
- Can you provide an example for the bonus? I actually don’t know what this means...

**Web Chef**
Homepage
- Omit the extra “to” in the pink callout box

**Story of Us**
Tell your story of self
- 2nd paragraph, 3rd sentence: “Where” should be “were”

Record and Reflect
- 2nd paragraph: “Storify allows you to collect content from across the web, just see this example.” Should it say “just like this example”?
- 2nd paragraph: The Greg McVerry link isn’t working properly – can’t click on it.
- Back to the index link is 404

**Comment by chadsansing** [comment]
Thanks, @KGorr - I think I got everything, but the index link will stay 404 until these pages are connected to the module landing page.

I feel like I need to go back and add some broswers.

**Comment by hannahkane** [comment]
Is this ticket still valid?

**Comment by chadsansing** [comment]
Probably not until we solve localization and the new template.

**Comment by flukeout** [comment]
What can I help with here @chadsansing ?

**Comment by chadsansing** [comment]
@flukeout - we should probably close this until there's a solution for localization that brings Web Lit I in-line w/ newer templates. As @ldecoursy points out, we'd likely break the localization on those activities if we change them now.
